### ISSUE TITLE
Not passing response object

### ISSUE THREAD

**Issue by E1cid** [ottoszika/node-red-contrib-ewelink#82]
Hi and once again thank you for your efforts

I am raising this issue as the device nodes does not pass the response object from the http in node to the http response node. This causes a error in the http response node. It would be better if the ewelink device node forwarded on all msg elements not just the payload.

**Comment by ottoszika** [comment]
Hello @E1cid,

You can transform your data as you like before linking to **HTTP Response** node. Just use a **function** node for this.

Regards,
Otto

**Comment by E1cid** [comment]
yes i can ,  but i had already transformed my data and passed it through the device node, and msg.res and msg.req get deleted. Nothing gets passed, the device node deletes all, and create msg.payload. It would be better for your node to pass all other msg elements on. I could pass them around your node and join the object but make the flow very clutter when you use 12 device nodes,.
